Data/ByteString/Char8.hs

```diff
@@ -284,7 +283,7 @@ pack = packChars
 {-# INLINE pack #-}
 
 -- | /O(n)/ Converts a 'ByteString' to a 'String'.
-unpack :: ByteString -> [Char]
+unpack :: ByteString -> String
```

Comment:
Normally String for [Char] is sensible, but here spelling it out makes sense given what it's doing and the context.